python/core/auto_generated/locator/qgslocatorfilter.sip.in

```diff
@@ -252,6 +252,14 @@ Tests a ``candidate`` string to see if it should be considered a match for
 a specified ``search`` string.
 Filter subclasses should use this method when comparing strings instead
 of directly using QString.contains() or Python 'in' checks.
+%End
+
+    static double fuzzyScore( const QString &candidate, const QString &search );
```

Comment:
Instead of adding this as a new method, could you extend the existing "stringMatches" function with an optional flag to enable fuzzy matching?

Reply:
How do I obtain the score value then, which is needed for sorting? I either change the return type or pass by reference, both sound bad to me.

Reply:
Fair point !